<?xml version="1.0" encoding="UTF-8" ?>
<testsuites disabled="0" errors="0" failures="1" tests="2" time="8.556">
 <testsuite name="demo calculator test" timestamp="2022-02-18T18:20:31" hostname="localhost" time="8.556" errors="0" tests="2" skipped="0" disabled="0" failures="1">
  <testcase classname="demo calculator test" name="addititon test" time="8.396" />
  <testcase classname="demo calculator test" name="Substraction test" time="0.152">
   <failure type="exception" message="Failed: homepage.clickButton1 is not a function"><![CDATA[TypeError: homepage.clickButton1 is not a function
    at UserContext.<anonymous> (C:\Users\Prathamesh\Desktop\ProtractorProject\ProtractorDemo\Tests\calculator.js:39:18)
    at C:\Users\Prathamesh\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:112:25
    at new ManagedPromise (C:\Users\Prathamesh\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1077:7)
    at ControlFlow.promise (C:\Users\Prathamesh\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2505:12)
    at schedulerExecute (C:\Users\Prathamesh\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:95:18)
    at TaskQueue.execute_ (C:\Users\Prathamesh\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
    at TaskQueue.executeNext_ (C:\Users\Prathamesh\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
    at C:\Users\Prathamesh\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27
    at C:\Users\Prathamesh\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
From: Task: Run it("Substraction test") in control flow
    at UserContext.<anonymous> (C:\Users\Prathamesh\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:94:19)
    at attempt (C:\Users\Prathamesh\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\Prathamesh\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at QueueRunner.execute (C:\Users\Prathamesh\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4199:10)
    at Spec.queueRunnerFactory (C:\Users\Prathamesh\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:909:35)
    at Spec.execute (C:\Users\Prathamesh\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:526:10)
    at UserContext.fn (C:\Users\Prathamesh\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:5340:37)
    at attempt (C:\Users\Prathamesh\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\Prathamesh\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at runNext (C:\Users\Prathamesh\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4257:20)
From asynchronous test: 
Error
    at Suite.<anonymous> (C:\Users\Prathamesh\Desktop\ProtractorProject\ProtractorDemo\Tests\calculator.js:27:5)
    at addSpecsToSuite (C:\Users\Prathamesh\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1107:25)
    at Env.describe (C:\Users\Prathamesh\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1074:7)
    at describe (C:\Users\Prathamesh\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4399:18)
    at Object.<anonymous> (C:\Users\Prathamesh\Desktop\ProtractorProject\ProtractorDemo\Tests\calculator.js:3:1)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)]]>
   </failure>
  </testcase>
 </testsuite>
</testsuites>